### DISCUSSION TITLE
About 3d_registration tutorial

### DISCUSSION THREAD

**Discussion by yeahtaiga** [Project-MONAI/MONAI#3441] [Q&A]
Hello. I'm learning about 3d medical image registration at https://github.com/Project-MONAI/tutorials/tree/master/3d_registration .
In this tutorial, why are some fixed_image missing in the section of nference with the pretrained model?

**Comment by wyli** [comment]
It just sampled very sparsely a few slices of the same indices of the input volumes, the goal is to present some basic qualitative results. cc @kate-sann5100

**Reply by yeahtaiga** [reply]
So is "fixed_image" not a single real MR image?

**Reply by wyli** [reply]
It is a volume during inference,the final visualisation is with the sampled 2d slices.
